admin/class-yoast-form.php

```diff
@@ -153,25 +153,17 @@ public function set_options_value( $key, $value, $overwrite = false ) {
 	 */
 	public function admin_footer( $submit = true, $show_sidebar = true ) {
 		if ( $submit ) {
-			$floating_button = new WPSEO_Admin_Saved_Changes_Listener();
+			$settings_changed_listener = new WPSEO_Admin_Settings_Changed_Listener();
```

Comment:
Can't help but feel a bit conflicted about having to instantiate this class just to echo a static message. The `have_settings_been_saved` there is also just a return of a static value.
Until the dependency injection, this might still be better than actual static, not sure. Did you think about this?